src/Surfnet/StepupMiddleware/ApiBundle/Tests/Service/DeprovisionServiceTest.php

```diff
@@ -106,4 +108,37 @@ public function test_it_can_return_data()
         $this->assertTrue(is_array($data));
         $this->assertEquals($data['status'], 'OK');
     }
+
+    public function test_deprovision_does_not_deprovision_when_user_is_not_found()
+    {
+        $this->apiRepo
+            ->shouldReceive('findOneByNameId')
+            ->with('urn:collab:person:example.com:maynard_keenan')
+            ->once()
+            ->andReturnNull();
+        $this->pipeline
+            ->shouldNotHaveReceived('process');
+        $this->deprovisionService->deprovision('urn:collab:person:example.com:maynard_keenan');
+    }
+    public function test_deprovision_method_performs_the_right_to_be_forgotten_command()
```

Comment:
The test does not perform any assertions, this leads to an incomplete test.
Is there a reason for this?

Reply:
The test `test_deprovision_does_not_deprovision_when_user_is_not_found` indeed does not have a traditional PHPUnit assertion. Yielding the test warning. 

I thought I remedied this by adding the m::close() method call in the teardown method. This is not complete. I updated this by using the `MockeryPHPUnitIntegration` trait.